lib/hpke/src/certificate.cpp

```diff
@@ -246,6 +294,18 @@ Certificate::subject() const
   return parsed_cert->subject;
 }
 
+std::string
+Certificate::subject_common_name() const
```

Comment:
Let's not destructure like this; instead just return the `ParsedName` representing the subject / issuer.